Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "alloy-mev"
-version = "0.2.1"
+version = "0.3.0"
```

Comment:
Can you change the version to 0.2.2 please? As there are no breaking change from 0.2.1

Reply:
I put it to 0.3.0 because Alloy was updated from 0.2.1 to 0.3.0 yesterday - since alloy-mev is intended to be used with Alloy, I thought that was reasonable due to the breaking changes in Alloy. Happy to change it to 0.2.2 if you disagree though 

https://crates.io/crates/alloy/versions

Reply:
Yes please :)

Reply:
Done :)